src/index.html

```diff
@@ -11,7 +11,204 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="set">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Name:
+          </label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            required
+          >
```

Comment:
you need to wrap input in label like you did for other elements, or add for attribute